iOS/Flows/Message Detail/MessageDetailViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  MessageDetailViewController.swift
+//  Jibber
+//
+//  Created by Martin Young on 2/24/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+@MainActor
+protocol MessageDetailViewControllerDelegate: AnyObject {
+    func messageDetailViewController(_ controller: MessageDetailViewController,
+                                     didSelectThreadFor message: Messageable)
+}
+
+class MessageDetailViewController: ViewController {
+
+    let message: Messageable
+    unowned let delegate: MessageDetailViewControllerDelegate
+
+    private let textView = TextView(font: .regular, textColor: .T3)
+    private let bubbleView = MessageBubbleView(orientation: .down, bubbleColor: ThemeColor.D1.color)
```

Comment:
I'm manually creating the text view and bubble instead of using the message content view. To use the message content view we'll need to make it easier to enable interaction. Wouldn't be a bad idea, but I'm holding off for now.